RELEASE_NOTES.md

```diff
@@ -0,0 +1,59 @@
+
+# # v20230201002.0.0
+
+## Notes
+
+ The Atlas Golang SDK official v20230201002.0.0 release. 
+
+## Breaking Changes
+
+1. All golang structures starting from `admin.Nullable` have been removed.
+Structures weren't part of the API and should not be used by end users.
+2. Data structures now use golang base types instead of `NullableString` or `NullableInt`.
+For example:
+
+`ThridPartyIntegration.ChannelName NullableString` => `ChannelName *string`
```

Comment:
A few comments:
Use unordered lists instead.
Start all items in the relnotes with a verb. 
Q: what do you mean by the phrase "starting from ..." Should it instead be: "Starting with"? I suggested this change but please see if it makes sense.
Suggested edit:

- Removed all internal golang structures starting with `admin.Nullable`. Data structures now use golang base 
   types instead of `NullableString` or `NullableInt`. For example, the data structure
  `ThridPartyIntegration.ChannelName NullableString` now uses the `ChannelName *string` base type.

Reply:
Awesome feedback! thank you so much @JuliaMongo. Will use that from now on.